framework/arcane-framework/.idea/modules/arcane-framework.arcane-framework-build.iml

```diff
@@ -195,4 +195,9 @@
       </library>
     </orderEntry>
   </component>
+  <component name="SbtModule">
```

Comment:
I believe this is missing from .gitignore. Please fix

Reply:
done